teams/developer-relations/bible-for-developers/general-items-breakdown.md

```diff
@@ -0,0 +1,51 @@
+# General Items Breakdown
+
+### General Items \(Applies to All Stages\)
+
+* [ ] High-schooler friendly writing style, easy to understand
+* [ ] Correct numbering
+* [ ] Every card has code\* 
+* [ ] 1 scroll per card\*\*
+* [ ] Proper grammar, punctuation, capitalization, etc.
+* [ ] Acceptable Styling per the Pull Request Checklist
+
+\*Exception being hard and medium cards in labs as well as activity cards that just introduce concepts
+
+\*\* If you have to scroll more than once to view the whole card, the card is too long
+
+Before going into a stage-by-stage breakdown, here's a general overview of things to keep in mind.
+
+#### Writing Style
+
+First and foremost, we want all our developers to use a conversational writing style when making cards. This is because the target audience for our curriculum is high school students, and our goal is to make technical education as accessible as possible. Writing as if you're having a conversation will keep students more engaged. When in doubt, think **blog post** instead of **calculus textbook**.
+
+#### Numbering
+
+It's crucial that each card is numbered correctly so developers can keep lessons organized and students can follow a logical progression. BitProject uses two numbering schemes, one for Activities and one for Labs.
+
+Here's an example of an Activity \(pretty self-explanatory\):
+
+![Numbering for Activity Cards](../../../.gitbook/assets/screen-shot-2020-03-29-at-3.00.05-pm.png)
+
+And here's an example of a Lab:
+
+![Numbering for Lab Cards](../../../.gitbook/assets/screen-shot-2020-03-29-at-3.06.54-pm.png)
```

Comment:
This is just me being nitpicky, But I think it might be better to show the example after you explained the numbering scheme. Also, I don't think the self-explanatory in parenthesis part is necessary. I realize you meant it for the activity cards, cuz it is self explanatory for that part. But I initially misunderstood that you meant that it was self-explanatory for the lab cards as well(which it isn't lol), and this could confuse beginners.

Reply:
thanks, i'll definitely take that into consideration!